lms/templates/edxnotes/edxnotes.html

```diff
@@ -0,0 +1,93 @@
+                <div class="tab-list is-hidden">
+                    <h2 id="tab-view" class="tabs-label">${_('View notes by:')}</h2>
+                </div>
+                <div class="inline-error is-hidden" tabindex="-1"></div>
+            % if notes:
+                <div class="ui-loading" tabindex="-1">
+                    <p><span class="spin"><i class="icon-refresh"></i></span> <span class="copy">${_("Loading...")}</span></p>
+                </div>
+            % else:
+                <section class="is-empty">
+                    <article>
+                        <h2>${_('You have not made any notes in this course yet.')}</h2>
+                        <p>${_('Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.')}</p>
+
+                        <h2>${_('Other Students In This Course Are Using Notes To:')}</h2>
+                        <ul>
+                            <li>${_('Add study notes to course content for exams and homework.')}</li>
+                            <li>${_('Highlight important concepts for later coursework or future courses')}</li>
+                            <li>${_('Donec id elit non mi porta gravida et eget metus.')}</li>
+                            <li>${_('Praesent commodo cursus magna, vel scelerisque nisl consectetur et.')}</li>
+                        </ul>
+
+                    % if position is not None:
+                        <div class="student-notes-cta">
+                            <h3 class="sr">${_('Start creating notes')}</h3>
+                            <p>${_('Get started by making a note in something you just read, like {section_link}.').format(
+                                section_link='<a href="{url}">{section_name}</a>'.format(
+                                    url=position['url'],
+                                    section_name=position['display_name'],
+                                )
+                            )}</p>
+                        </div>
+                    % endif
+                    </article>
+                </section>
+            % endif
+            </section>
+        </section>
+    </div>
+</section>
+
+## Include Underscore templates
+<%block name="header_extras">
+% for template_name in ["note-item", "tab-item"]:
+<script type="text/template" id="${template_name}-tpl">
+  <%static:include path="edxnotes/${template_name}.underscore" />
+</script>
+% endfor
+</%block>
+<%block name="js_extra">
+% if notes:
+<script type="text/javascript">
+    (function (require) {
+        require(['js/edxnotes/views/page_factory'], function (NotesFactory) {
```

Comment:
@andy-armstrong 
>I'd call this NotesPageFactory, since this isn't a factory that creates notes.

Changed `NotesFactory` to `NotesPageFactory`. Left the file name unchanged though. @polesye FYI.